Sources/App/add.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  remote.swift
+//  Guaka
+//
+//  Created by Omar Abdelhafith on 05/11/2016.
+//
+//
+
+import Guaka
+
+var addCommand: Command = Command(
+  name: "add", 
+  run: executeAddCommand, parent: rootCommand)
```

Comment:
Just brainfarting here could we also have an API that in the constructor receives a closure? that would be slick.